ci_framework/roles/edpm_build_images/tasks/run.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Set DIB elements path
+  ansible.builtin.set_fact:
+    cifmw_elements: "{{ cifmw_edpm_build_images_elements | join(':') }}"
+    cacheable: true
+
+- name: Build EDPM hardened uefi image
+  when:
+    - (cifmw_edpm_build_images_hardened_uefi | bool) or (cifmw_edpm_build_images_all | bool)
+  environment:
+    DIB_LOCAL_IMAGE: "{{ cifmw_edpm_build_images_dib_local_image_path }}/{{ cifmw_edpm_build_images_base_image_name }}"
+    DIB_YUM_REPO_CONF: "{{ cifmw_edpm_build_images_dib_yum_repo_conf | join(' ') }}"
+    ELEMENTS_PATH: "{{ cifmw_elements }}"
+  ansible.builtin.shell: |
+    diskimage-builder images/edpm-hardened-uefi-centos-9-stream.yaml > {{ cifmw_edpm_build_images_artifacts }}/edpm_images/edpm_hardened_uefi_image_build.log
+  args:
+    chdir: "{{ cifmw_edpm_image_builder_repo_path }}"
```

Comment:
We can change directory to {{ cifmw_edpm_image_builder_repo_path }}/images and directly pass file file diskimage-builder images/edpm-hardened-uefi-centos-9-stream.yaml

 chdir: "{{ cifmw_edpm_image_builder_repo_path }}/images"